calculator/main.py

```diff
@@ -13,7 +13,13 @@ def create_new_calculator(operations=None):
     :param operations: Dict with initial operations.
                        ie: {'sum': sum_function, ...}
     """
-    pass
+    calculator = {'operations':{}, 'history':[]}
+    if operations is not None:
+        if not isinstance(operations, dict):
+            raise InvalidParams()
+        for key, value in operations.items():
```

Comment:
Thanks, Jason. I must admit this is the first time I see this method. Using it makes sense.